letsdrill.rb

```diff
@@ -1,24 +1,31 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  case integer
```

Comment:
Nice clean solution using the case statement!  One other thing to consider might be handling invalid grades.